feat(apple): Clarify bitcode is deprecated (#6757)
getsentry/sentry-docs: src/platform-includes/debug-symbols-apple/_default.mdx

<Note>

> Since April 25th, 2023, the [App Store](https://developer.apple.com/news/?id=jd9wcyov) doesn't accept submissions with bitcode enabled.

Reviewer: ```suggestion
As of April 25th, 2023, the [App Store](https://developer.apple.com/news/?id=jd9wcyov) no longer accepts submissions with bitcode enabled.
```
